svc-bie-kafka/docker-entryprep.sh

```diff
@@ -12,6 +12,13 @@ for ENV_VAR in \
   fi
 done
 
+export KEYSTORE_FILE="$PWD/keystore.p12"
+echo "$BIE_KAFKA_KEYSTORE_INBASE64" | base64 -d > "$KEYSTORE_FILE"
+echo -e "\nVerifying keystore ($KEYSTORE_FILE) and its password..."
+if ! keytool -list -v -keystore "$KEYSTORE_FILE" -storepass "$BIE_KAFKA_KEYSTORE_PASSWORD" | grep "Alias name:"; then
```

Comment:
I recognize that this if statement on line 18 is trying to test whether the password successfully unlocks the keystore, and I am wondering if we are able to use the command's error code to check for the pass/fail state in favor of the "Alias name" text.  If there is some reason that doesn't work in this case, it is fine the way it is, but I wanted to ask the question since it stood out as a potential improvement.

Reply:
The improvement could be to either implement RBAC changes to the mock Kafka cluster or find a way to ensure that the keystore attribute in the application.yaml file applies only to local and integration tests, without affecting other environments.